src/lib/components/Measure.react.js

```diff
@@ -0,0 +1,95 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import LeafletMeasure from '../LeafletMeasure';
+
+/**
+ * Measure is based on https://github.com/ljagis/leaflet-measure
+ */
+export default class Measure extends Component {
+
+    render() {
+        const nProps = Object.assign({}, this.props);
+        nProps.action = () => {
+            this.props.setProps({n_clicks: this.props.n_clicks + 1});
+        };
+        return <LeafletMeasure {...nProps}/>
+    }
+
+}
+
+Measure.defaultProps = {
+    n_clicks: 0
```

Comment:
I guess this code was just copied from the button example code? As i read it, it's not used by this component.